### PR TITLE
Added the logo to the docs

### LINKED ISSUE #25275: Add logo guide to the docs
We should have a page in the docs that just has our logo that people can copy, and a guide on how to use it. Right now the logo is available if you build it in the source, but it's not easily accessible from a Google search.

### PR DESCRIPTION
**Pull request by wermos**
#### References to other Issues or PRs

Closes #25275. 

#### Brief description of what is fixed or changed
Added the SymPy logo to the docs, under the "How-To Guides" section.

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by wermos** [comment]
I'm having a bit of trouble building the docs locally. When I run `make html` from `sympy/doc`, I get the following errors:
```
make -s _html || make -s printwarnings
Running Sphinx v5.3.0

Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/home/tirthankar/projects/sympy/sympy/__init__.py", line 22, in <module>
    import mpmath
ModuleNotFoundError: No module named 'mpmath'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/sphinx/config.py", line 350, in eval_config_file
    exec(code, namespace)
  File "/home/tirthankar/projects/sympy/doc/src/conf.py", line 22, in <module>
    import sympy
  File "/home/tirthankar/projects/sympy/sympy/__init__.py", line 24, in <module>
    raise ImportError("SymPy now depends on mpmath as an external library. "
ImportError: SymPy now depends on mpmath as an external library. See https://docs.sympy.org/latest/install.html#mpmath for more information.

make[1]: *** [Makefile:69: _html] Error 2
The docs build failed with the following errors:

Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/home/tirthankar/projects/sympy/sympy/__init__.py", line 22, in <module>
    import mpmath
ModuleNotFoundError: No module named 'mpmath'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/sphinx/config.py", line 350, in eval_config_file
    exec(code, namespace)
  File "/home/tirthankar/projects/sympy/doc/src/conf.py", line 22, in <module>
    import sympy
  File "/home/tirthankar/projects/sympy/sympy/__init__.py", line 24, in <module>
    raise ImportError("SymPy now depends on mpmath as an external library. "
ImportError: SymPy now depends on mpmath as an external library. See https://docs.sympy.org/latest/install.html#mpmath for more information.

make[1]: *** [Makefile:77: printwarnings] Error 1
make: *** [Makefile:58: html] Error 2
```
It claims that I don't have `mpmath` installed.

However, I _do_ have it installed, as can be seen from the output of `pip list`
```
❯ pip list
Package                Version   Editable project location
---------------------- --------- -------------------------------
antlr4-python3-runtime 4.11.1
iniconfig              2.0.0
jedi                   0.18.2
lark                   1.1.5
mpmath                 1.3.0
packaging              23.1
parso                  0.8.3
pip                    23.1.2
pluggy                 1.0.0
pudb                   2022.1.3
pydot                  1.4.2
Pygments               2.15.1
pyparsing              3.0.9
pytest                 7.3.1
setuptools             66.1.1
sympy                  1.13.dev0 /home/tirthankar/projects/sympy
urwid                  2.1.2
urwid-readline         0.13
```

I had the same issue after I did a fresh clone of SymPy, made a virtualenv, and ran `pip install mpmath` and `pip install -e .`.

**Comment by sylee957** [comment]
Where is the sphinx installed?
It is possible that sphinx got pulled from some other environment (like apt packages)

**Comment by asmeurer** [comment]
We should include the actual logo images on the page so that people can just copy them directly. 

Also it's not necessary to call this "press kit". I would just call it "SymPy Logo" and put it in one of the existing subdirectories like guides or contributing.

**Comment by asmeurer** [comment]
Another option would be to put this on the website instead of the docs.

**Comment by wermos** [comment]
> Where is the sphinx installed?

Thanks, that was the issue 🤦‍♂️

> We should include the actual logo images on the page so that people can just copy them directly.

I'll do that. I didn't see the messages on this PR until just now so the latest commit didn't incorporate any of those suggestions.

> Also it's not necessary to call this "press kit". I would just call it "SymPy Logo" and put it in one of the existing subdirectories like guides or contributing.

Understood. I think putting it by itself, or at least under "guides", makes more sense because using the SymPy Logo is orthogonal to contributing to the codebase. For putting it under "guides", we can rename it to something like "find the SymPy logo" to make it fit the mold of ["How-To guides"](https://docs.sympy.org/dev/guides/index.html).

On that note, I'm noticing that none of the items under "How-To guides" follow that mold so maybe just renaming it to "Guides" might be a good idea (not in this PR, obviously). I suppose there is no reason to name it "find the SymPy logo" in that case, and "SymPy Logo" by itself will suffice.

**Comment by wermos** [comment]
For now, I've put it in the `guides/` directory as @asmeurer suggested. If needed, we can always move it somewhere else later.

File: doc/src/guides/logo.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+We would like to make it easy for you to include the SymPy project identity in
+your next academic paper, course materials, or presentation.
+
+.. image::  ../logo/sympy.svg
+  :width: 600
+  :align: center
+  :alt: SymPy Logo
+
+You can find this high-resolution version of the SymPy logo
+`here <https://github.com/sympy/sympy/blob/master/doc/src/logo/sympy.svg>`_.
```
Comment (asmeurer) [inline doc/src/guides/logo.rst:14]: There's no reason to have this since the above image is basically the same thing.

**Comment by asmeurer** [comment]
This looks good. We should also include a PNG version as that may be what some people need. We can also include the notext version, with a note that the text version should be preferred unless the "SymPy" name is already present separately.

File: doc/src/guides/logo.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+If you would like to generate SymPy's collection of official logos yourself,
+you can do so by running:
+
+.. code-block:: none
+
+    $ cd doc
+
+    $ make logo # will be stored in the _build/logo subdirectory
```
Comment (asmeurer) [inline doc/src/guides/logo.rst:23]: We should mention that this requires the `rsvg-convert` function. We can cross-link to the docs dependencies page for instructions on how to install it.

**Comment by sylee957** [comment]
I don't think that we favor the binary files like `png` because of version controls.
If there is a way to do without it, it can be favored.

**Comment by asmeurer** [comment]
It's not necessary to commit the png file. It is generated with `make logo`, which is already run when the docs are built.

File: doc/src/guides/logo.rst (hunk 1 above)
Comment (wermos) [reply doc/src/guides/logo.rst:14]: Removed.

File: doc/src/guides/logo.rst (hunk 2 above)
Comment (wermos) [reply doc/src/guides/logo.rst:23]: Done.

**Comment by wermos** [comment]
> We should also include a PNG version as that may be what some people need. We can also include the notext version, with a note that the text version should be preferred unless the "SymPy" name is already present separately.

I've added both of these into the SymPy Logo page, along with the note.

> It's not necessary to commit the png file. It is generated with make logo, which is already run when the docs are built.

Thanks, I didn't know about that. I made sure to include the PNG files from the `_build/logo/`.

**Comment by asmeurer** [comment]
This looks good.

**Comment by wermos** [comment]
@asmeurer if you are fine with this, then I think it can be merged to `master` as it is passing all tests.

**Pull request merged**